protocol-units/bridge/chains/movement/src/lib.rs

```diff
@@ -395,22 +456,77 @@ impl BridgeContractCounterparty for MovementClient {
 		recipient: RecipientAddress<Self::Address>,
 		amount: Amount,
 	) -> BridgeContractCounterpartyResult<()> {
-		//@TODO properly return an error instead of unwrapping
+		
+		let test_noop_payload = utils::make_aptos_payload(
+			self.counterparty_address,
+			COUNTERPARTY_MODULE_NAME,
+			"noop_for_testing",
+			Vec::new(),
+			Vec::new(),
+		);
```

Comment:
Let's remove the `test_noop`, logic as that was just part of your debugging process. We don't need it in the branch.